app/views/hyrax/file_sets/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<%# OVERRIDE: Add in the :ext_relation to the FileSet edit page %>
+<%= simple_form_for [main_app, curation_concern], html: { multipart: true, class: 'nav-safety' } do |f| %>
+  <fieldset class="required">
+    <span class="control-label">
+      <%= label_tag 'file_set[title][]', t('.title'),  class: "string optional" %>
+    </span>
+    <%= text_field_tag 'file_set[title][]', curation_concern.title.first, class: 'form-control', required: true %>
+  </fieldset>
+
+  <br/>
+
+  <%# ADD: Create the :ext_relation field to edit on the FileSet %>
+  <% if !curation_concern.ext_relation.blank? %>
+    <fieldset class="required">
+      <span class="control-label">
+        <%= label_tag 'file_set[ext_relation]', t('.ext_relation'),  class: "string optional" %>
+      </span>
+      <%= text_field_tag 'file_set[ext_relation]', curation_concern.ext_relation, class: 'form-control', type: 'url', required: true %>
+    </fieldset>
+  <% end %>
+
+  <div class="row">
+    <div class="col-md-12 form-actions">
+      <%= f.submit(
```

Comment:
This looks a little wonky. Mostly layout wise. Can this get made easier to read?

Reply:
Let me reorganize it in a way it is easier to read